.github/workflows/publish_docs.yml

```
@@ -0,0 +1,17 @@
+name: 'Publish docs'
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material
+      - run: |
+          cd www
+          mkdocs gh-deploy --force
```

`insert_final_newline = true` 😊

```suggestion
          mkdocs gh-deploy --force

```